### PR TITLE
Support `UpCunet2x_Fast` for Real-CUGAN

### LINKED ISSUE #182: UpCunet2x_Fast support for ShuffleCugan
Hi, I was reading through the code and failed to see any mentions of shufflecugan, are there any plans to support sudo's shufflecugan model based on UpCunet2x_Fast?

https://openmodeldb.info/models/2x-sudo-shuffle-cugan-9-584-969

Amazing work so far guys, much appreciated!

### PR DESCRIPTION
**Pull request by RunDevelopment**
Resolves #182

This adds support for `UpCunet2x_fast`, also known as ShuffleCugan, under Real-CUGAN architecture. The google drive file I use for testing is a re-upload of [`sudo_shuffle_cugan_9.584.969`](https://openmodeldb.info/models/2x-sudo-shuffle-cugan-9-584-969) to my google drive.

### CODE REVIEW

File: libs/spandrel/spandrel/architectures/RealCUGAN/arch/upcunet_v3.py [hunk 1]
```diff
@@ -419,3 +419,55 @@ def forward(self, x: Tensor, alpha: float = 1):
             x = (x - 0.15) / 0.7
 
         return x
+
+
+class pixel_unshuffle(nn.Module):
```
Comment (joeyballentine) [inline libs/spandrel/spandrel/architectures/RealCUGAN/arch/upcunet_v3.py:424]: I know it's just because it was in the original code, but why on earth wouldn't they just use torch.nn.pixelshuffle??

Comment (joeyballentine) [reply libs/spandrel/spandrel/architectures/RealCUGAN/arch/upcunet_v3.py:424]: In fact, this arch could probably be sped up by just swapping that out

Comment (RunDevelopment) [reply libs/spandrel/spandrel/architectures/RealCUGAN/arch/upcunet_v3.py:424]: This might have been a compatibility thing to support older pytorch versions, idk. I just tested it with `nn.PixelUnshuffle` and the output is the same. So I agree that we should use the torch version.

**Review by joeyballentine — APPROVED** [review]
